api/swagger/swagger-v1.yaml

```diff
@@ -2045,6 +2045,106 @@ paths:
         "500":
           description: Server error
           content: {}
+
+  /users/{id}/coins:
+    get:
+      tags:
+      - users
+      description: 'Gets a list of the coins owned by the user and their balances'
+      parameters:
+      - name: id
+        in: path
+        description: A User ID
+        required: true
+        schema:
+          type: string
+      operationId: Get User Coins
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: object 
+                properties:
+                  data:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        mint:
+                          type: string
+                          description: The coin mint address
+                          example: "9LzCMqDgTKYz9Drzqnpgee3SGa89up3a247ypMj2xrqM"
+                        ticker:
+                          type: string
+                          description: The coin symbol
+                          example: "$AUDIO"
+                        balance:
+                          type: integer
+                          description: The balance of the coin in the user's account (in wei)
+                          example: 1000000000
+                        balance_usd:
+                          type: number
+                          description: The balance of the coin in the user's account in USD
+                          example: 1.23
+  /users/{id}/coins/{mint}:
+    get:
+      tags:
+      - users
+      description: 'Gets information about a specific coin owned by the user and their wallets'
+      parameters:
+      - name: id
+        in: path
+        description: A User ID
+        required: true
+        schema:
+          type: string
+      operationId: Get User Coin
+      responses:
+        '200':
```

Comment:
I go back and forth on whether it's useful to include the possible 4xx/5xx responses in these specs.